src/openzaak/components/zaken/tests/test_migrations.py

```diff
@@ -155,46 +159,52 @@ def test_composite_urls_filled(self):
         self.zio_known.refresh_from_db()
         self.zaak_besluit_known.refresh_from_db()
 
-        self.assertEqual(self.zaak_known._zaaktype_base_url, self.ztc_known)
+        self.assertEqual(self.zaak_known._zaaktype_base_url.pk, self.ztc_known.pk)
```

Comment:
Essentially what is described here: https://github.com/open-formulieren/open-forms/pull/4758#discussion_r1821323098

Since I think we're only interested here in checking if the right service is linked, I figured it would be enough to compare the PK. What do you think?

Reply:
TIL! Sure, let's use PKs here